urukul.py

```diff
@@ -94,27 +96,31 @@ class CFG(Module):
     | DIV       | 2     | Clock divider configuration: 0: default,        |
     |           |       | 1: divide-by-one, 2: divider-by-two,            |
     |           |       | 3: divide-by-four                               |
+    | ATT_EN    | 4     | Enable ATT (per channel)                        |
+    | DUMMY     | 1     | Unused, not usable, undefined                   |
```

Comment:
Is it necessary to define dummy bits at the end?

Reply:
No, this was tested and they have been removed.